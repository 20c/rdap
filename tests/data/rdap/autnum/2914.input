{
  "rdapConformance": [
    "nro_rdap_profile_0",
    "rdap_level_0",
    "nro_rdap_profile_asn_flat_0"
  ],
  "notices": [
    {
      "title": "Terms of Service",
      "description": [
        "By using the ARIN RDAP/Whois service, you are agreeing to the RDAP/Whois Terms of Use"
      ],
      "links": [
        {
          "value": "https://rdap.arin.net/registry/autnum/2914",
          "rel": "terms-of-service",
          "type": "text/html",
          "href": "https://www.arin.net/resources/registry/whois/tou/"
        }
      ]
    },
    {
      "title": "Whois Inaccuracy Reporting",
      "description": [
        "If you see inaccuracies in the results, please visit: "
      ],
      "links": [
        {
          "value": "https://rdap.arin.net/registry/autnum/2914",
          "rel": "inaccuracy-report",
          "type": "text/html",
          "href": "https://www.arin.net/resources/registry/whois/inaccuracy_reporting/"
        }
      ]
    },
    {
      "title": "Copyright Notice",
      "description": [
        "Copyright 1997-2023, American Registry for Internet Numbers, Ltd."
      ]
    }
  ],
  "handle": "AS2914",
  "startAutnum": 2914,
  "endAutnum": 2914,
  "name": "NTT-LTD-2914",
  "remarks": [
    {
      "title": "Registration Comments",
      "description": [
        "Website: https://www.gin.ntt.net",
        "Routing Policy: https://www.gin.ntt.net/support-center/policies-procedures/routing/"
      ]
    }
  ],
  "events": [
    {
      "eventAction": "last changed",
      "eventDate": "2021-11-24T11:59:32-05:00"
    },
    {
      "eventAction": "registration",
      "eventDate": "1998-12-07T00:00:00-05:00"
    }
  ],
  "links": [
    {
      "value": "https://rdap.arin.net/registry/autnum/2914",
      "rel": "self",
      "type": "application/rdap+json",
      "href": "https://rdap.arin.net/registry/autnum/2914"
    },
    {
      "value": "https://rdap.arin.net/registry/autnum/2914",
      "rel": "alternate",
      "type": "application/xml",
      "href": "https://whois.arin.net/rest/asn/AS2914"
    }
  ],
  "entities": [
    {
      "handle": "NTTAM-1",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "fn",
            {},
            "text",
            "NTT America, Inc."
          ],
          [
            "adr",
            {
              "label": "15809 Bear Creek Pkwy\nSuite 320\nRedmond\nWA\n98052\nUnited States"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "kind",
            {},
            "text",
            "org"
          ]
        ]
      ],
      "roles": [
        "registrant"
      ],
      "links": [
        {
          "value": "https://rdap.arin.net/registry/autnum/2914",
          "rel": "self",
          "type": "application/rdap+json",
          "href": "https://rdap.arin.net/registry/entity/NTTAM-1"
        },
        {
          "value": "https://rdap.arin.net/registry/autnum/2914",
          "rel": "alternate",
          "type": "application/xml",
          "href": "https://whois.arin.net/rest/org/NTTAM-1"
        }
      ],
      "events": [
        {
          "eventAction": "last changed",
          "eventDate": "2023-02-06T13:00:49-05:00"
        },
        {
          "eventAction": "registration",
          "eventDate": "2005-12-08T12:04:23-05:00"
        }
      ],
      "entities": [
        {
          "handle": "NAAC-ARIN",
          "vcardArray": [
            "vcard",
            [
              [
                "version",
                {},
                "text",
                "4.0"
              ],
              [
                "adr",
                {
                  "label": "8300 E Maplewood Ave.\nSuite 400\nGreenwood Village\nCO\n80111\nUnited States"
                },
                "text",
                [
                  "",
                  "",
                  "",
                  "",
                  "",
                  "",
                  ""
                ]
              ],
              [
                "fn",
                {},
                "text",
                "NTT America Abuse Contact"
              ],
              [
                "org",
                {},
                "text",
                "NTT America Abuse Contact"
              ],
              [
                "kind",
                {},
                "text",
                "group"
              ],
              [
                "email",
                {},
                "text",
                "abuse@ntt.net"
              ],
              [
                "tel",
                {
                  "type": [
                    "work",
                    "voice"
                  ]
                },
                "text",
                "+1-877-688-6625"
              ]
            ]
          ],
          "roles": [
            "abuse"
          ],
          "links": [
            {
              "value": "https://rdap.arin.net/registry/autnum/2914",
              "rel": "self",
              "type": "application/rdap+json",
              "href": "https://rdap.arin.net/registry/entity/NAAC-ARIN"
            },
            {
              "value": "https://rdap.arin.net/registry/autnum/2914",
              "rel": "alternate",
              "type": "application/xml",
              "href": "https://whois.arin.net/rest/poc/NAAC-ARIN"
            }
          ],
          "events": [
            {
              "eventAction": "last changed",
              "eventDate": "2023-01-19T16:36:41-05:00"
            },
            {
              "eventAction": "registration",
              "eventDate": "2005-12-02T11:16:33-05:00"
            }
          ],
          "status": [
            "validated"
          ],
          "port43": "whois.arin.net",
          "objectClassName": "entity"
        },
        {
          "handle": "NASC-ARIN",
          "vcardArray": [
            "vcard",
            [
              [
                "version",
                {},
                "text",
                "4.0"
              ],
              [
                "adr",
                {
                  "label": "8300 E Maplewood Ave.\nSuite 400\nGreenwood Village\nCO\n80111\nUnited States"
                },
                "text",
                [
                  "",
                  "",
                  "",
                  "",
                  "",
                  "",
                  ""
                ]
              ],
              [
                "fn",
                {},
                "text",
                "NTT America Support Contact"
              ],
              [
                "org",
                {},
                "text",
                "NTT America Support Contact"
              ],
              [
                "kind",
                {},
                "text",
                "group"
              ],
              [
                "email",
                {},
                "text",
                "support@us.ntt.net"
              ],
              [
                "tel",
                {
                  "type": [
                    "work",
                    "voice"
                  ]
                },
                "text",
                "+1-877-688-6625"
              ]
            ]
          ],
          "roles": [
            "noc"
          ],
          "links": [
            {
              "value": "https://rdap.arin.net/registry/autnum/2914",
              "rel": "self",
              "type": "application/rdap+json",
              "href": "https://rdap.arin.net/registry/entity/NASC-ARIN"
            },
            {
              "value": "https://rdap.arin.net/registry/autnum/2914",
              "rel": "alternate",
              "type": "application/xml",
              "href": "https://whois.arin.net/rest/poc/NASC-ARIN"
            }
          ],
          "events": [
            {
              "eventAction": "last changed",
              "eventDate": "2023-01-19T16:36:02-05:00"
            },
            {
              "eventAction": "registration",
              "eventDate": "2005-12-02T11:16:39-05:00"
            }
          ],
          "status": [
            "validated"
          ],
          "port43": "whois.arin.net",
          "objectClassName": "entity"
        },
        {
          "handle": "VIPAR-ARIN",
          "vcardArray": [
            "vcard",
            [
              [
                "version",
                {},
                "text",
                "4.0"
              ],
              [
                "adr",
                {
                  "label": "8300 E Maplewood Ave.\nSuite 400\nGreenwood Village\nCO\n80111\nUnited States"
                },
                "text",
                [
                  "",
                  "",
                  "",
                  "",
                  "",
                  "",
                  ""
                ]
              ],
              [
                "fn",
                {},
                "text",
                "VIPAR"
              ],
              [
                "org",
                {},
                "text",
                "VIPAR"
              ],
              [
                "kind",
                {},
                "text",
                "group"
              ],
              [
                "email",
                {},
                "text",
                "ipr@gin.ntt.net"
              ],
              [
                "tel",
                {
                  "type": [
                    "work",
                    "voice"
                  ]
                },
                "text",
                "+1-877-688-6625"
              ]
            ]
          ],
          "roles": [
            "administrative",
            "technical"
          ],
          "links": [
            {
              "value": "https://rdap.arin.net/registry/autnum/2914",
              "rel": "self",
              "type": "application/rdap+json",
              "href": "https://rdap.arin.net/registry/entity/VIPAR-ARIN"
            },
            {
              "value": "https://rdap.arin.net/registry/autnum/2914",
              "rel": "alternate",
              "type": "application/xml",
              "href": "https://whois.arin.net/rest/poc/VIPAR-ARIN"
            }
          ],
          "events": [
            {
              "eventAction": "last changed",
              "eventDate": "2022-12-06T14:32:05-05:00"
            },
            {
              "eventAction": "registration",
              "eventDate": "2005-12-02T11:16:27-05:00"
            }
          ],
          "status": [
            "validated"
          ],
          "port43": "whois.arin.net",
          "objectClassName": "entity"
        },
        {
          "handle": "CANDE70-ARIN",
          "vcardArray": [
            "vcard",
            [
              [
                "version",
                {},
                "text",
                "4.0"
              ],
              [
                "adr",
                {
                  "label": "101 Park Ave.\n41st. floor\nNew York\nNY\n10178\nUnited States"
                },
                "text",
                [
                  "",
                  "",
                  "",
                  "",
                  "",
                  "",
                  ""
                ]
              ],
              [
                "fn",
                {},
                "text",
                "Massimo Candela"
              ],
              [
                "n",
                {},
                "text",
                [
                  "Candela",
                  "Massimo",
                  "",
                  "",
                  ""
                ]
              ],
              [
                "kind",
                {},
                "text",
                "individual"
              ],
              [
                "email",
                {},
                "text",
                "massimo@ntt.net"
              ],
              [
                "tel",
                {
                  "type": [
                    "work",
                    "voice"
                  ]
                },
                "text",
                "+1-214-915-1366"
              ]
            ]
          ],
          "roles": [
            "technical"
          ],
          "links": [
            {
              "value": "https://rdap.arin.net/registry/autnum/2914",
              "rel": "self",
              "type": "application/rdap+json",
              "href": "https://rdap.arin.net/registry/entity/CANDE70-ARIN"
            },
            {
              "value": "https://rdap.arin.net/registry/autnum/2914",
              "rel": "alternate",
              "type": "application/xml",
              "href": "https://whois.arin.net/rest/poc/CANDE70-ARIN"
            }
          ],
          "events": [
            {
              "eventAction": "last changed",
              "eventDate": "2023-02-06T12:21:20-05:00"
            },
            {
              "eventAction": "registration",
              "eventDate": "2023-02-06T12:21:20-05:00"
            }
          ],
          "status": [
            "validated"
          ],
          "port43": "whois.arin.net",
          "objectClassName": "entity"
        }
      ],
      "port43": "whois.arin.net",
      "objectClassName": "entity"
    },
    {
      "handle": "PEERI-ARIN",
      "vcardArray": [
        "vcard",
        [
          [
            "version",
            {},
            "text",
            "4.0"
          ],
          [
            "adr",
            {
              "label": "101 Park Ave.\r\n41st. floor\nNew York\nNY\n10178\nUnited States"
            },
            "text",
            [
              "",
              "",
              "",
              "",
              "",
              "",
              ""
            ]
          ],
          [
            "fn",
            {},
            "text",
            "Peering"
          ],
          [
            "org",
            {},
            "text",
            "Peering"
          ],
          [
            "kind",
            {},
            "text",
            "group"
          ],
          [
            "email",
            {},
            "text",
            "peering@ntt.net"
          ],
          [
            "tel",
            {
              "type": [
                "work",
                "voice"
              ]
            },
            "text",
            "+1-877-688-6625"
          ]
        ]
      ],
      "roles": [
        "technical"
      ],
      "links": [
        {
          "value": "https://rdap.arin.net/registry/autnum/2914",
          "rel": "self",
          "type": "application/rdap+json",
          "href": "https://rdap.arin.net/registry/entity/PEERI-ARIN"
        },
        {
          "value": "https://rdap.arin.net/registry/autnum/2914",
          "rel": "alternate",
          "type": "application/xml",
          "href": "https://whois.arin.net/rest/poc/PEERI-ARIN"
        }
      ],
      "events": [
        {
          "eventAction": "last changed",
          "eventDate": "2023-02-27T09:10:43-05:00"
        },
        {
          "eventAction": "registration",
          "eventDate": "2006-09-05T19:36:06-04:00"
        }
      ],
      "status": [
        "validated"
      ],
      "port43": "whois.arin.net",
      "objectClassName": "entity"
    }
  ],
  "port43": "whois.arin.net",
  "status": [
    "active"
  ],
  "objectClassName": "autnum"
}