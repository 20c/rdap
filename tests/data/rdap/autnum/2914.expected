{"org_address": "8300 E Maplewood Ave.\r\nSuite 400\nGreenwood Village\nCO\n80111\nUNITED STATES", "org_name": "NTT America, Inc.", "emails": ["abuse@ntt.net", "peering@ntt.net", "support@us.ntt.net", "vipar@us.ntt.net"], "name": "NTT-COMMUNICATIONS-2914"}
